next.config.js

```diff
@@ -37,7 +37,13 @@ const withTypedCssModules = config => {
 module.exports = withPWA({
   reactStrictMode: true,
   images: {
-    domains: ["drive.google.com", "acmucsd.s3.us-west-1.amazonaws.com", "i.imgur.com"],
+    domains: [
+      "drive.google.com",
+      "acmucsd.s3.us-west-1.amazonaws.com",
+      "i.imgur.com",
+      "media.licdn.com",
+      "mojeanmac.github.io"
```

Comment:
This is fine, but we should really try to move away from letting people provide images from their own domain. I think storing profile pics in s3 should be our future goal, then we don't need to keep adding domains